tools/cldr-apps/src/main/java/org/unicode/cldr/web/UserRegistry.java

```diff
@@ -2453,6 +2461,7 @@ int writeUserFile(SurveyMain sm, String ourDate, boolean obscured, File outFile)
                 rs = ps.executeQuery();
                 if (rs == null) {
                     out.println("\t<!-- No results -->");
+                    out.close();
```

Comment:
This fixes a compiler warning -- unrelated to this ticket but same file and too small for its own ticket